app/views/travel_requests/_review_form.html.erb

```diff
@@ -74,6 +74,7 @@
           <card-content>
             <data-table
               caption="Anticipated Expenses"
+              class="estimates"
```

Comment:
This line should be combined with the already existing class on line 79

Reply:
Removed the overriding class